### PR TITLE
Docker Tags in line with ark-analysis Version

### LINKED ISSUE #634: Latest tag is referencing the 3.6 version
**Describe the bug**
The `latest` tag is referencing the 3.6 version, not 3.7, so "/usr/local/lib/python3.6" is still created. When importing modules, Python is looking inside of the python3.6 folder first (so the up-to-date code in python3.7 is not being imported). 

**To Reproduce**
Re-build the docker and import ark module.

**Expected behavior**
`latest` tag should reference 3.7 version.

### PR DESCRIPTION
**Pull request by srivarra**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Closes #634, #659.

Currently `latest` always points to the most recent build of the docker image, which may not be the current version of the repository. As a best practice we should tag the docker image to the same version of ark-analysis.

**How did you implement your changes**

Adjusted the docker build section of the `.travis.yml` to include the tag, and pushing it properly.

**Remaining issues**

None atm.

### CODE REVIEW

**Comment by srivarra** [comment]
@ngreenwald The readthedocs issue is being fixed in #650. The docker does get tagged with `docker-test`. So it seems to work.

File: .travis.yml [hunk 1]
```diff
@@ -41,11 +41,10 @@ jobs:
...
       script:
         - "travis_wait 120 sleep 7200 &"
-        - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin 
-        - docker build -t "$TRAVIS_REPO_SLUG" . 1> /dev/null
-        - docker tag "$TRAVIS_REPO_SLUG" "$TRAVIS_REPO_SLUG":latest
-        - docker tag "$TRAVIS_REPO_SLUG" "$TRAVIS_REPO_SLUG":"${TRAVIS_TAG}"
-        - docker push "$TRAVIS_REPO_SLUG"
+        - echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+        - docker build -t "$TRAVIS_REPO_SLUG":"$TRAVIS_TAG" . 1> /dev/null
+        # Tag the image with a version like v0.3.2, and create another tag for latest
```
Comment (ngreenwald) [inline .travis.yml:46]: This comment should be updated I think

File: Dockerfile [hunk 2]
```diff
@@ -5,6 +5,10 @@ RUN apt-get update
 
 # install dependencies needed for setting up R
 RUN apt-get install -y lsb-release dirmngr gnupg apt-transport-https ca-certificates software-properties-common
+RUN apt-get install -y libharfbuzz-dev libfribidi-dev
+
+# RUN apt-get -y build-dep libcurl4-gnutls-dev
```
Comment (ngreenwald) [inline Dockerfile:10]: Do we need this here still?

File: Dockerfile [hunk 3]
```diff
@@ -23,6 +27,17 @@ RUN apt-get install -y cmake
 
 WORKDIR /scripts
 
+# Install R dependency packages
+RUN R -e "install.packages('arrow')"
+RUN R -e "install.packages('data.table')"
+RUN R -e "install.packages('doParallel')"
+RUN R -e "install.packages('foreach')"
```
Comment (ngreenwald) [inline Dockerfile:34]: Did moving the order of R vs python resolve the issues?

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Just some small stuff

File: Dockerfile (hunk 2 above)
Comment (srivarra) [reply Dockerfile:10]: No, I will get rid of that commented bit.

File: Dockerfile (hunk 3 above)
Comment (srivarra) [reply Dockerfile:34]: No, however it's a quality of life change. The Docker builds the Dockerfile line by line, and we do not expect the R packages to be changed very often if at all. If Docker does not notice changes based on a line, then it'll skip it. However the Python stuff will always be changing, so it'll cut development build time.

**Review by ngreenwald — APPROVED** [review]
Looks good! Can you put out a new release to make sure everything is working as expected?

**Pull request merged**